2023 Istio Steering Contributor Seats (#12580)
istio/istio.io: content/en/blog/2023/steering-contribution-seat-results/index.md

---
title: "2023 Istio Steering Contributor Seats"
subtitle: Announcing the Istio Steering Committee Contributor Members, 2023.
description: Announcing the Istio Steering Committee Contributor Members, 2023.
publishdate: 2023-02-04
attribution: "Faseela K, for the Istio Steering Committee"
keywords: [istio,steering,governance,community,contributor]
---

Based on our [Steering Committee charter](/blog/2020/steering-changes/), Istio shares governance responsibilities between Contribution Seats, selected based on contributions to the project, and Community Seats, elected by the project members. The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 proportionally-allocated Contribution Seats, and 4 elected Community Seats.

> Last year, we had [elected four members](/blog/2022/steering-election-results/) to the community seats, with the committee representing eight different companies. It's now time to reward the companies who fuel our growth with maximum contribution to Istio, by selecting the [Contributor seat](/blog/2020/steering-changes/#contribution-seats) members to the Istio Steering Committee. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), the Contribution Seat numbers is calculated in February, and the last agreed [metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md) for the same is Contributions, as measured by [devstats](https://istio.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All).

Reviewer: ```suggestion
Last year, we [elected four members](/blog/2022/steering-election-results/) to the community seats, with the committee representing eight different companies. It's now time to reward the companies who fuel our growth with the maximum contributions to Istio, by selecting the [Contributor seat](/blog/2020/steering-changes/#contribution-seats) members to the Istio Steering Committee. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), the Contribution Seat numbers are calculated in February, and the last agreed [metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md) is Contributions, as measured by [devstats](https://istio.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All).
```

> According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. As a result we have the following new members joining the Istio steering, in addition to the existing contributor members.

Reviewer: ```suggestion
According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. As a result we have the following new members joining the Istio Steering Committee, in addition to the existing contributor members.
```

---
title: "2023 Istio Steering Contributor Seats"
subtitle: Announcing the Istio Steering Committee Contributor Members, 2023.
description: Announcing the Istio Steering Committee Contributor Members, 2023.
publishdate: 2023-02-04
attribution: "Faseela K, for the Istio Steering Committee"
keywords: [istio,steering,governance,community,contributor]
---

> Based on our [Steering Committee charter](/blog/2020/steering-changes/), Istio shares governance responsibilities between Contribution Seats, selected based on contributions to the project, and Community Seats, elected by the project members. The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 proportionally-allocated Contribution Seats, and 4 elected Community Seats.

Reviewer 2: ```suggestion
The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 Contribution Seats, proportionally allocated based on corporate contributions to the project, and 4 elected Community Seats.
```

Last year, we [elected the four members](/blog/2022/steering-election-results/) to the community seats. It's now time to reward the companies who fuel our growth by selecting the [Contribution Seat] members. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), every February we look at which companies have made the most contributions to Istio based on an [annually agreed metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md).

> According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. As a result we have the following new members joining the Istio Steering Committee, in addition to the existing contributor members.

Reviewer 2: ```suggestion
According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. As a result we have the following new members joining the Istio Steering Committee:
```

> Last year, we [elected four members](/blog/2022/steering-election-results/) to the community seats, with the committee representing eight different companies. It's now time to reward the companies who fuel our growth with the maximum contributions to Istio, by selecting the [Contributor seat](/blog/2020/steering-changes/#contribution-seats) members to the Istio Steering Committee. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), the Contribution Seat numbers are calculated in February, and the last agreed [metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md) is Contributions, as measured by [devstats](https://istio.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All).

Reviewer 2: "reward" is a strange term to use.  (I think we normally also say 7, counting IBM/Red Hat as one group.)

```suggestion
Last year, we [elected the four members](/blog/2022/steering-election-results/) to the community seats. It's now time to reward the companies who fuel our growth by selecting the [Contribution Seat] members. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), every February we look at which companies have made the most contributions to Istio based on an [annually agreed metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md).
```

> According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. As a result we have the following new members joining the Istio Steering Committee, in addition to the existing contributor members.

Reviewer 2: Should we just list all the contributor members at this point?

Author: Yes, listed everyone's name

Author: Fixed the order. @ericvn . Also pushed a PR in community to fix the order along with adding new members : https://github.com/istio/community/pull/969

- [Cale Rath](https://github.com/ctrath) (IBM)
- [Justin Pettit](https://github.com/justinpettit) (Google)
- [John Howard](https://github.com/howardjohn) (Google)
- [Zhonghu Xu](https://github.com/hzxuzhonghu) (Huawei)

> We wish to offer our thanks to Srihari Angaluri, Kebe Liu and Louis Ryan, all long-time contributors to the Istio project, whose term will be ending now.

Reviewer: We should add Jason here as well.

Author: Done

> We wish to offer our thanks to Srihari Angaluri, Kebe Liu and Louis Ryan, all long-time contributors to the Istio project, whose term will be ending now.

Reviewer 2: All three leaving for different reasons.  Does saying "long-time contributors" add anything? 

```suggestion
Our sincerest thanks to Srihari Angaluri, Kebe Liu and Louis Ryan, all long-time contributors to the Istio project, whose terms have come to an end.
```

---

> title: "2023 Istio Steering Contributor Seats"
> subtitle: Announcing the Istio Steering Committee Contributor Members, 2023.
> description: Announcing the Istio Steering Committee Contributor Members, 2023.
> publishdate: 2023-02-04

Reviewer 2: ```suggestion
title: "Announcing the Contribution Seat holders for 2023"
description: The Istio Steering Committee welcomes contributors from Google, IBM, Huawei and Red Hat.
publishdate: 2023-02-06
```

attribution: "Faseela K, for the Istio Steering Committee"
keywords: [istio,steering,governance,community,contributor]
---

The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 Contribution Seats, proportionally allocated based on corporate contributions to the project, and 4 elected Community Seats.

> Last year, we [elected the four members](/blog/2022/steering-election-results/) to the community seats. It's now time to announce the companies who fuel our growth by selecting the [Contribution Seat] members. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), every February we look at which companies have made the most contributions to Istio based on an [annually agreed metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md).

Reviewer: elected ~~the~~ four

Reviewer: Do we want []'s around contribution seat? It seems we are missing a link?

Author: i think it was a merging mistake. fixed.

---
title: "Announcing the Contribution Seat holders for 2023"
description: The Istio Steering Committee welcomes contributors from Google, IBM, Huawei and Red Hat.
publishdate: 2023-02-06
attribution: "Faseela K, for the Istio Steering Committee"
keywords: [istio,steering,governance,community,contributor]
---

The [Istio Steering Committee](https://github.com/istio/community/blob/master/steering/README.md) consists of 9 Contribution Seats, proportionally allocated based on corporate contributions to the project, and 4 elected Community Seats.

Last year, we [elected four members](/blog/2022/steering-election-results/) to the community seats. It's now time to announce the companies who fuel our growth by selecting the Contribution Seat members. As per the [Steering charter](https://github.com/istio/community/blob/master/steering/CHARTER.md), every February we look at which companies have made the most contributions to Istio based on an [annually agreed metric](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md).

> According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. 

Reviewer 2: ```suggestion
According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats.
```

Reviewer 2: markdown lint makes me sad.  trailing spaces don't offend anyone!

Reviewer: > markdown lint makes me sad. trailing spaces don't offend anyone!

We could put an exception in the config if it makes sense to do so.

Reviewer 2: It wouldn't generally be a big deal, because I would normally have access to accept my own or other people's suggestions/edit a PR before it went live, and I would just fix it.  I can't do that with Faseela's, for some reason. (Perhaps I am not as much of an admin as I was once.)

Can you accept the change?

Reviewer: I didn't see the option to accept yours, so tried creating one of my own to see about accepting that. Doesn't appear to be an option anymore. I suspect there might have been some tightening of things as I noted I couldn't push a commit on another PR which I thought I could do earlier. It's probably a good idea to not allow anyone to necessarily push PRs. Might also be a setting on the fork?

> According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats. 

Reviewer: ```suggestion
According to our [seat allocation process](https://docs.google.com/spreadsheets/d/1IIms6OT1DgJ_lbn5433sy5wvqNBIIQgQHoWZvpShSXk/edit#gid=1365082320), this year Google will be allocated 5 seats and IBM/Red Hat will be allocated 2. As the third largest contributor to Istio in the last 12 months, we are pleased to announce that Huawei has earned two Contribution Seats.
```

Reviewer 2: I don't have access to accept this change.  As a maintainer, I normally do.  I think there's a checkbox for "allow edits from maintainers", which is usually ticked, and may not be for Faseela's PRs?
